scripts/openstack-quickstart-demosetup

```diff
@@ -264,7 +264,8 @@ EODASHDB
 SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = USE_SSL
 EOSEC
     # sync dashboard DB "after" the database is created
-    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' syncdb --noinput"
+    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' makemigrations"
```

Comment:
Ah sorry for misunderstand your comment, but I am not sure that this is correct (SOC7):

```
# psql horizon
horizon=# \d

                      List of relations
 Schema |             Name              |   Type   |  Owner  
--------+-------------------------------+----------+---------
 public | auth_group                    | table    | horizon
 public | auth_group_id_seq             | sequence | horizon
 public | auth_group_permissions        | table    | horizon
 public | auth_group_permissions_id_seq | sequence | horizon
 public | auth_permission               | table    | horizon
 public | auth_permission_id_seq        | sequence | horizon
 public | django_content_type           | table    | horizon
 public | django_content_type_id_seq    | sequence | horizon
 public | django_migrations             | table    | horizon
 public | django_migrations_id_seq      | sequence | horizon
 public | django_session                | table    | horizon
 public | openstack_auth_user           | table    | horizon
(12 rows)
```